### PR TITLE
Feat: OS CLI output update

### PR DESCRIPTION
**Pull request by dagrest**
- [ ] Follows [CONTRIBUTING](https://github.com/snyk/snyk/blob/master/CONTRIBUTING.md) rules

#### What does this PR do?

- add description labels
- add issue summary
- actionable focus on upgrade
- count issues
- newline separates issues
- update severity headline

This PR is based on the following PRs:
[feat:OS CLI output part I - update severity headline +test](https://github.com/snyk/cli/pull/3437)
[feat:OS CLI output part II - newline separates issues +test](https://github.com/snyk/cli/pull/3470)
[feat:OS CLI output part III - count issues +test](https://github.com/snyk/cli/pull/3499)
[feat:OS CLI output part IV - actionable focus on upgrade](https://github.com/snyk/cli/pull/3500)
[feat:OS CLI output part V - add issue summary](https://github.com/snyk/cli/pull/3560)
[feat:OS CLI output part VI - add description labels](https://github.com/snyk/cli/pull/3563)

#### Where should the reviewer start?

src/lib/formatters/remediation-based-format-issues.ts
src/lib/formatters/format-test-meta.ts
src/lib/formatters/types.ts

#### How should this be manually tested?

Run 'snyk test' and 'snyk test --unmanaged' commands on a local project and compare output formatting with the attached screenshot

#### What are the relevant tickets?

What are the relevant tickets?
[TUN-269](https://snyksec.atlassian.net/browse/TUN-269)

#### Additional questions

[TUN-269]: https://snyksec.atlassian.net/browse/TUN-269?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/lib/formatters/test/format-test-results.ts [hunk 1]
```diff
@@ -221,12 +233,70 @@ export function getDisplayedOutput(
         )
       : [];
 
+  const licenseBySeverityText = `${Object.keys(
```
Comment (dekelund) [inline src/lib/formatters/test/format-test-results.ts:236]: It feels a bit strange use of backticks here (line 236, 248 and line 254), at least it's hard to follow where strings start and end.

Comment (dekelund) [reply src/lib/formatters/test/format-test-results.ts:236]: ah it's also in line 250. as I said it's hard to follow

Comment (dagrest) [reply src/lib/formatters/test/format-test-results.ts:236]: It's ok since that it’s within a ${}-expression

**Review by dekelund — DISMISSED** [review]
